delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -110,20 +140,20 @@ node {
 
 			if (fileExists('build.api')) {
 				flowType = "API"
-				//events.setServiceType("API")
+				events.setServiceType("API")
 				loadServerlessConfig(runtime, configPackBBURL)
 				//changeServerlessConfig(runtime)
 				//updateServiceNameConfig(domain, service_name)
 				updateSwaggerConfig(domain, params.service_name.trim(), var_api_key, config)
 			} else if(fileExists('build.lambda')) {
 				flowType = "LAMBDA"
-				//events.setServiceType("LAMBDA")
+				events.setServiceType("LAMBDA")
```

Comment:
ServiceType should be "FUNCTION", This is a change pending in internal version. We can address this in the OSS version right away.
Make these enumerations, so you dont have to hardcode strings. You can change the enumeration in one place if you have to in the future.

Reply:
Did the changes. Made the changes for servrcetype to be added as FUNCTION